### PR TITLE
Port Medibot Tipping!

### PR DESCRIPTION
**Pull request by Rykka-Stormheart**
Port of https://github.com/Citadel-Station-13/Citadel-Station-13/pull/12537

Ethics memes aside, the idea off people being caught off guard at their guilt for causing harm to inanimate objects is really funny to me, and feels extra relevant to SS13. Considering people already regularly get in fights and maim each other over harming station pets, I picture this being a great source of arguments and fights ICly over what torturing a cute lil' medbot says about you.

tl;dr this PR lets you tip over medbots which prevents them from moving or healing, and they'll slowly get more and more distressed and desperate asking for help from people around them. After a few minutes, if no one helps right them, they'll manage to right themselves up and snitch on whoever tipped them over to medical so the psychologist can keep an eye on them.

### CODE REVIEW

**Comment by KasparoVy** [comment]
`Considering people already regularly get in fights and maim each other over harming station pets`
S'truth! Where I come from that's a declaration of war

**Comment by Rykka-Stormheart** [comment]
> Touches Polaris file. Probably should be ported to Polaris.

Already is. https://github.com/PolarisSS13/Polaris/pull/7356

**Comment by dwinters99** [comment]
This is the most important PR I've ever seen. I am in love with this. Why _are_ we still here? _**Just to suffer?**_
